src/hn/app.py

```diff
@@ -46,19 +47,18 @@ def __init__(self, *args, **kwargs):
         self.old_child = None
         self.setup_styles()
         self.set_default_size(360, 720)
-        self.show_all()
 
         Bus.on("open_thread", self.set_thread)
         Bus.on("open_website", self.set_website)
 
     def setup_styles(self):
         css_provider = Gtk.CssProvider()
-        context = Gtk.StyleContext()
-        screen = Gdk.Screen.get_default()
-
         css_provider.load_from_resource('/hn/css/style.css')
```

Comment:
If you use `AdwApplication` and gresources you can cut this segment out entirely.